packages/client/src/createClient.ts

```diff
@@ -54,6 +54,19 @@ class ActionInvoker<Q extends ActionDefinition<any, any, any>>
   applyAction: (...args: any[]) => any;
 }
 
+class QueryInvoker<Q extends QueryDefinition<any, any>>
```

Comment:
Basically kept in parity with what you did for actions

Reply:
Sweet. Thank you